File: Javascript/tower.js
function towerBuilder(nFloors) {
  return Array.from({length: nFloors}, (_, i) => ' '.repeat(nFloors - i - 1) + '*'.repeat(i * 2 + 1) +  ' '.repeat(nFloors - i - 1))
}

const { assert } = require("chai")

describe("Tests", () => {
  it("test", () => {
    assert.deepEqual(towerBuilder(1), ["*"]);
    assert.deepEqual(towerBuilder(2), [" * ","***"]);
    assert.deepEqual(towerBuilder(3), ["  *  "," *** ","*****"]);
  });
});

⬇️⬇️

/* Example (nFloors = 3)：
[
  '  *  ', 
  ' *** ', 
  '*****'
]
*/

File: python/tower.py
def tower_builder(n_floors):
    return [' ' * (n_floors - i - 1) + '*' * (i * 2 + 1) + ' ' * (n_floors - i - 1) for i in range(n_floors)]
  
import codewars_test as test
from solution import tower_builder

@test.describe("Build Tower")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(tower_builder(1), ['*', ])
        test.assert_equals(tower_builder(2), [' * ', '***'])
        test.assert_equals(tower_builder(3), ['  *  ', ' *** ', '*****'])
        
⬇️⬇️

"""Example (n_floors = 3)：
[
  '  *  ', 
  ' *** ', 
  '*****'
]
"""
